pypesto/logging.py

```diff
@@ -7,51 +7,63 @@
 
 import logging
 
-
-def log_to_console(level=None):
-    """
-    Log to console.
+def log(name: str = 'pypesto',
+        level: int = None,
+        console: bool = False,
+        filename: str = ''):
+    '''
```

Comment:
please use triple double quotes (""") here. don't know if that is pep8, but in general we use double quotes to indicate "human-readable" text vs. single quotes for identifiers etc.